python/ray/util/tracing/enable_tracing.py

```diff
@@ -0,0 +1,19 @@
+from opentelemetry import trace
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.exporter.otlp.proto.grpc.trace_exporter import OTLPSpanExporter
+from opentelemetry.sdk.trace.export import (
+    ConsoleSpanExporter,
+    SimpleSpanProcessor,
+)
+from typing import Any
+
+
+def setup_tracing(*args: Any, **kwargs: Any) -> None:
+    # Sets the tracer_provider. This is only allowed once per execution
+    # context and will log a warning if attempted multiple times.
+    trace.set_tracer_provider(TracerProvider())
+    trace.get_tracer_provider().add_span_processor(
+        SimpleSpanProcessor(
+            OTLPSpanExporter(endpoint="http://localhost:4317", insecure=True)))
```

Comment:
Oh I see this is the specific oltp exporter. Can we put this in a different name in tests?